app/_locales/en/messages.json

```diff
@@ -2330,6 +2330,9 @@
   "origin": {
     "message": "Origin"
   },
+  "osTheme": {
+    "message": "System Preference"
```

Comment:
Is this copy something we have spoken to design about? 

Since I last commented, I observed another website using "Light", "Dark", and "System". I think "System" rather than "System Preference" could be a nice option here. It might pan out better with translations as well.

Reply:
I think `System` should be intuitive enough, and be easily translatable.